backend/src/main/java/quarano/actions/DescriptionCode.java

```diff
@@ -15,6 +15,8 @@
 
 	DIARY_ENTRY_MISSING(true),
 
+	QUARANTINE_ENDING(true),
```

Comment:
Not sure about this one 🤔 Is it manual resolvable? I guessed `yes` because the case can be concluded if the `TrackedPerson` is symptom free out of the quarantine, right?